src/fusion_power_plant.h

```diff
@@ -106,6 +106,16 @@ class FusionPowerPlant : public cyclus::Facility  {
   }
   double sequestered_equilibrium; 
 
+  #pragma cyclus var { \
+    "default": 0.9, \
+    "doc": "Fraction of desired startup tritium inventory required. ", \
```

Comment:
```suggestion
    "doc": "Fraction of desired tritium reserve inventory required for initial startup. ", \
```

Reply:
The actual implementation multiplies the entire startup (sequestered_equilibrium + reserve) by the fraction, not just the reserve, which is why I said "desired startup tritium" instead of "... reserve ...". Let me know if that needs to change.

Reply:
Yeah... but the user never actually declares a "startup" inventory in the input file.  Maybe this should be:
```suggestion
    "doc": "Fraction of desired startup ( = reserve + sequestered) tritium inventory required for initial startup. ", \
```

It also reads oddly because why would you not need 100% of the startup inventory for startup?  If you don't actually need that much for startup, why call it the startup inventory.... maybe its just a vocabulary thing?

Reply:
That works, I'll commit! Just wanted to make sure we were being clear.